pkg/kopia/command/common.go

```diff
@@ -26,6 +26,14 @@ type CommandArgs struct {
 	LogDirectory   string
 }
 
+type KopiaCommandParams struct {
```

Comment:
We can just use the new struct `GeneralCommandArgs` now instead of these two separate structs.

Reply:
Okay will make changes !

Reply:
Made the changes !